### PR TITLE
Putting batch_axis,seq_axis instead batch_dim, seq_dim according

### PR DESCRIPTION
**Pull request by dimaxano**
According to [this](https://github.com/yongtang/tensorflow/blob/6a5d1dd1fb9fdea12f4b3572e9e94fed05245d15/tensorflow/python/ops/array_ops.py#L2622)

### CODE REVIEW

File: tensorflow/python/ops/rnn.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def _reverse(input_, seq_lengths, seq_dim, batch_dim):
       if seq_lengths is not None:
         return array_ops.reverse_sequence(
             input=input_, seq_lengths=seq_lengths,
-            seq_dim=seq_dim, batch_dim=batch_dim)
+            seq_axis=seq_dim, batch_axis=batch_dim)
```
Comment (qlzh727) [inline tensorflow/python/ops/rnn.py]: Also please rename variables from seq_dim to seq_axis, batch_dim to batch_axis accordingly. Thanks.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/rnn.py (hunk 1 above)
Comment (dimaxano) [reply tensorflow/python/ops/rnn.py]: Done!

**Comment by dimaxano** [comment]
Btw, what about changing time_dim, batch_dim variables [L420](https://github.com/dimaxano/tensorflow/blob/e8c44d765146d228ee88e46b1e1f1e8fb3894818/tensorflow/python/ops/rnn.py#L420)

**Comment by qlzh727** [comment]
That will be ideal too, thanks

**Comment by dimaxano** [comment]
Completed

**Comment by dimaxano** [comment]
Hi, @qlzh727. Is it everything correct with this PR?

**Review by qlzh727 — APPROVED** [review]

**Comment by qlzh727** [comment]
Sorry for the late reply. LGTM.

**Pull request merged**